package.json

```
@@ -7,7 +7,7 @@
     "@edx/edx-proctoring": "^1.5.0",
     "@edx/frontend-component-cookie-policy-banner": "1.0.0",
     "@edx/paragon": "2.6.4",
-    "@edx/studio-frontend": "^1.17.0",
+    "@edx/studio-frontend": "1.17.0",
```

We have a new version [1.19.1](https://github.com/openedx/studio-frontend/commits/v1.19.1) with the fix and other updates.

I've tested with `maple.3` and tutor `13.2.2`.

```suggestion
    "@edx/studio-frontend": "1.19.1",
```

---

Sorry, I just committed the new version.